### ISSUE TITLE
Remove parms library

### ISSUE THREAD

**Issue by JanStreffing** [FESOM/fesom2#569]

**Comment by sebastianbeyer** [comment]
The PR is against master, not refactoring, should we update that?

**Comment by JanStreffing** [comment]
you are right. I see what happens when I fix that. might be easier to start over again from refactoring tough.

**Comment by JanStreffing** [comment]
I closed https://github.com/FESOM/fesom2/pull/582. To be redone.
